go.mod

```diff
@@ -71,6 +71,8 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.2.0
 )
 
+require github.com/simplesurance/bunny-go v0.0.0-20221115111006-e11d9dc91f04
```

Comment:
can you move this element into the previous `require` block?

Reply:
Sure, didn't see GoLand put that in its own block. Weird